projects/wave-core-new/src/lib/operators/dialogs/temporal-raster-mean-plot-dialog/temporal-raster-mean-plot-dialog.component.html

```diff
@@ -0,0 +1,45 @@
+<wave-sidenav-header>Temporal Raster Mean Plot</wave-sidenav-header>
+
+<wave-dialog-help>
+    <p>
+        This operator generates a <i>Temporal Raster Mean Plot</i> based on a raster data set (<i>Input</i>).
+        Select the <i>Time Tick Position</i>, which specifies the position of the x-axis ticks,
+        e.g. at the the time start, end or center of the raster's validity.
+        The plot shows an area chart of the mean values (y-axis) of the rasters.
+        The x-axis displays the timestamps.
+        Edit the <i>Output Name</i> of the plot, if desired, and click on <i>Create</i> to generate the plot.
+    </p>
+    <p>
+        Note: Plots change dynamically with the map view and include only pixels or features, which are located in this area.
```

Comment:
maybe add that the plot only makes sense when selecting a time range?

Reply:
3328cae